src/main/resources/swagger/listings/workspaces

```diff
@@ -9,7 +9,7 @@
 			"method": "GET",
 			"summary": "Get workspace ACL",
 			"notes": "",
-			"type": "string",
+			"type": "WorkspaceACL",
```

Comment:
where exactly were you trying to add an enum? I might be able to help with that...